setup-ci/setup-git.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script sets up personal Git credentials for use in a Compute Instance and clones a repository. 
+# Adjust as needed.
+
+# parameters
+USERNAME="user"
+EMAIL_ADDRESS="user@contoso.com"
+PAT="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+REPO="github.com/contoso/repo.git"
+LOCAL_DIR="repo"
+
+# clone repository
+git clone https://$USERNAME:$PAT@$REPO $LOCAL_DIR
+
+# change into repository
+cd $LOCAL_DIR
+
+# configure global git on the Compute Instance
```

Comment:
Add a comment here.. that this will be saved on your computer instance local disk and NOT the shared file systems... You can find it in ~ from the terminal on CI